python/pyspark/worker.py

```diff
@@ -128,6 +128,17 @@ def wrapped(*series):
     return lambda *a: (wrapped(*a), arrow_return_type)
 
 
+def wrap_window_agg_pandas_udf(f, return_type):
+    arrow_return_type = to_arrow_type(return_type)
+
+    def wrapped(*series):
+        import pandas as pd
+        result = f(*series)
+        return pd.Series([result]).repeat(len(series[0]))
```

Comment:
So, this place is the only place where it's diverted (by repeating?); therefore, needs Windows specific attribute?

Reply:
Yes - I tried to do this on the Java side but it's tricky and complicated to merging the input row and output of udf if they are not 1-1 mapping. So I ended up doing this..